proto/pbevent/subscribe.proto

```diff
@@ -0,0 +1,119 @@
+/*
+Package event provides a service for subscribing to state change events.
+*/
+syntax = "proto3";
+
+package event;
```

Comment:
I'm not sure about this package name. Maybe subscribe or subscription would be better.

There are a lot of events (serf UserEvent, audit events), so maybe subscribe would help differentiate this from other changes.

Reply:
renamed to `pbsubscribe`